lib/class-wp-theme-json.php

```diff
@@ -485,12 +436,14 @@ private static function remove_keys_not_in_schema( $tree, $schema ) {
 		$tree = array_intersect_key( $tree, $schema );
 
 		foreach ( $schema as $key => $data ) {
-			if ( is_array( $schema[ $key ] ) && isset( $tree[ $key ] ) ) {
+			if ( is_array( $schema[ $key ] ) && is_array( $tree[ $key ] ) ) {
```

Comment:
`isset` accounts also for the case when the `$key` in the `$tree` is not set. Is it safe to replace it with `is_array`?

Reply:
It's not, good catch! Pushed another check above as well.

Would you know how to make unit tests fail if they are warnings? Somehow they're not reported for me when running `npm run test-unit-php`.

Reply:
Is it possible that the env that runs phpunit has error reporting disabled?